src/js/index.jsx

```diff
@@ -1,19 +1,28 @@
 import ReactDOM from 'react-dom';
 import React from 'react';
+import { Provider } from 'react-redux';
+import { createStore, applyMiddleware } from 'redux';
 import { BrowserRouter, Switch } from 'react-router-dom';
+import ReduxPromise from 'redux-promise';
 
 import 'bootstrap/dist/js/bootstrap';
 import 'bootstrap/dist/css/bootstrap.css';
 import '../css/main.scss';
 
+import reducers from './reducers';
```

Comment:
I think it would be better to have here same naming convention as in /reducers/index.jsx. Other than that it looks ok.